src/models/pispTransaction.model.ts

```diff
@@ -82,6 +84,7 @@ export class PISPTransactionModel
       transitions: [
         // Party Lookup Phase
         { name: 'requestPartyLookup', from: 'start', to: 'partyLookupSuccess' },
+        { name: 'failPartyLookup', from: 'partyLookupSuccess', to: 'partyLookupFailure' },
```

Comment:
`partyLookupSuccess` going to `partyLookupFailure` looks a little odd to me. Should it be `partyLookupPending` or something instead of success?

Reply:
there could be a failure when there is no party found